### ISSUE TITLE
classifychars: Can not change selection in "Selection dashboard"

### ISSUE THREAD

**Issue by maechler** [openalpr/openalpr#506]
In the selection dashboard it is not possible to change the selection to a different binarization than the first one using left and right arrow keys. The green selection stays always as in the following image:

<img width="735" alt="screen shot 2017-05-16 at 21 37 12" src="https://cloud.githubusercontent.com/assets/4113819/26139001/bfa4bc7a-3acd-11e7-8d04-f92cb93975b9.png">

However when I select the first binarization and enter the character selector the left and right arrow keys work correctly. So I guess the key mappings should be fine.

I have installed openalpr using `brew install openalpr` and use the following versions: `openalpr 2.3.0` and `macOS 10.12.4`

I have also tried it on `Ubuntu 16.04` and `openalpr 2.2.4` installed with `apt-get`. The dashboard selection does not work either, in addition I can not even enter the character selection there.

**Question**
We managed to finish a training on 130 license plates resulting in around 700 characters, but got very poor results. That is way we figured that probably we should classify all the 3 binarization versions of one license plate in order to improve the results.
Can you tell us if that is the intention of these 3 binarizations? The docs are not clear about that.

**Comment by maechler** [comment]
@matthill Could you tell us whether that ought to work or not?

**Comment by ashish1405** [comment]
In OSx for us ctrl+c and ctrl+b is the key to select different binarization

**Comment by maechler** [comment]
Thank you very much for your answer! However that seems rather counter intuitive..

Do you also know whether one should create character tiles from all the binarizations?

**Comment by ashish1405** [comment]
Well AFAIK it is not necessary, but you can select which is visually best to you

**Comment by maechler** [comment]
@matthill Is there a place where this navigation by CTRL+C and CTRL+B could / should be documented? Or would it be possible to change it to navigation by arrow keys?

**Comment by ashish1405** [comment]
Yes it is possible, you will have to change code. Check lines 41-73. Based on you system you can set the value to the desired key. For checking what is the key, cout waitkey in both main and showCharSelection.

**Comment by maechler** [comment]
Would it make sense to change this in the repository? Or would this break the navigation for other Mac users?

I just feel like it should be the same keys in the `Selection dashboard` and the `Character selection`, at the moment this is not the case for me. In the `Character selection` it works using the arrow keys in the `Selection dashboard` I have to use CTRL+C and CTRL+B.
I am afraid I am not very experienced in C programming and such low level platform differences.

**Comment by ashish1405** [comment]
Well that may be due to the fact that in showCharSelection, the waykey is typecasted to char `if ((char) waitkey == LEFT_ARROW_KEY)`

**Comment by maechler** [comment]
@ashish1405 Thanks very much! When I find the time I will test this and create a pull request.
